.next/build-manifest.json

```diff
@@ -15,16 +13,7 @@
     "static/chunks/main-app.js"
   ],
   "pages": {
-    "/_app": [
-      "static/chunks/webpack.js",
-      "static/chunks/main.js",
-      "static/chunks/pages/_app.js"
-    ],
-    "/_error": [
-      "static/chunks/webpack.js",
-      "static/chunks/main.js",
-      "static/chunks/pages/_error.js"
-    ]
+    "/_app": []
   },
   "ampFirstPages": []
 }
```

Comment:
.next folder should not be committed, it should be in the gitignore file.

Reply:
Ok sir